packages/quick_blue/android/src/main/kotlin/com/example/quick_blue/QuickBluePlugin.kt

```diff
@@ -260,6 +262,13 @@ class QuickBluePlugin: FlutterPlugin, MethodCallHandler, EventChannel.StreamHand
 
     override fun onCharacteristicWrite(gatt: BluetoothGatt?, characteristic: BluetoothGattCharacteristic, status: Int) {
       Log.v(TAG, "onCharacteristicWrite ${characteristic.uuid}, ${characteristic.value.contentToString()} $status")
+      val deviceID = gatt?.device?.address ?: ""
+      // TODO Parse status code and send correct error messages
+      sendMessage(messageConnector, mapOf(
+        "deviceId" to deviceID,
+        "characteristic" to characteristic.uuid.toString(),
+        "error" to "Write Command Status : $status"
```

Comment:
`"Write Command Status : $status"` is a error message, not a program recongized code

An `ErrorCode` with optional `ErrorMessage` is a better choice